boa_engine/src/object/jsmap.rs

```diff
@@ -0,0 +1,426 @@
+//! This module implements a wrapper for the Map Builtin Javascript Object
+use crate::{
+    builtins::map::{add_entries_from_iterable, ordered_map::OrderedMap},
+    builtins::Map,
+    object::{JsFunction, JsMapIterator, JsObject, JsObjectType, ObjectData},
+    Context, JsResult, JsValue,
+};
+
+use boa_gc::{Finalize, Trace};
+use std::ops::Deref;
+
+/// `JsMap` provides a wrapper for Boa's implementation of the Javascript `Map` object.
+///
+/// # Examples
+///
+/// Create a `JsMap` and set a new entry
+/// ```
+/// use boa_engine::{
+///    object::JsMap,
+///    Context, JsValue,
+/// };
```

Comment:
```suggestion
/// # use boa_engine::{
/// #     object::JsMap,
/// #     Context, JsValue,
/// # };
```

I'd recommend hiding the imports to have a cleaner example on each doc. Also applies to every example

Reply:
Agreed! Went through and hid the imports to clean up the examples. :)